### PR TITLE
feat(CB2-9696): GitHub Action for blocking merging on failing check

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## GitHub Action for running integration tests

Added IVA defects holding branch to the GitHub action to block merging if workflow fails
[link to ticket number](https://dvsa.atlassian.net/browse/CB2-9696)

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by LGin-BJSS — APPROVED** [review]

File: .github/workflows/pr-checks.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ name: PR-checks
 
 on:
   push:
-    branches: ["develop", "feature/VTMDEV-1"]
+    branches: ["develop", "feature/VTMDEV-1", "feature/CB2-9571"]
```
Comment (cb-cs) [inline .github/workflows/pr-checks.yml:8]: Have approved this, but may want to put a ticket in the backlog to undo this change once the holding branch has been merged into develop.

**Review by cb-cs — APPROVED** [review]
